sdk/src/main/java/com/uid2/UID2Manager.kt

```diff
@@ -309,6 +326,19 @@ public class UID2Manager internal constructor(
         }
     }
 
+    /**
+     * After initialization is complete, all the attached listeners will be invoked.
+     */
+    private suspend fun onInitialized() {
+        initializedLock.withLock {
+            onInitializedListeners.forEach { listener ->
+                listener()
+            }
+
+            onInitializedListeners.clear()
```

Comment:
up to you an absolutely preferential:

you could make the list to be a Deque (like with a LinkedList) and remove stuff when deleting it. Something like
```
while (onInitializedListeners.isNotEmpty()) {
  onInitializedListeners.removeFirst().invoke()
}
```

Reply:
Nice, will apply. Thanks!